project/RELEASE-CHECKLIST.md

```diff
@@ -49,7 +49,17 @@ git cherry-pick <commit-id>
 ...
 ```
 
-### 2. Update CHANGELOG.md
+### 2. Bump the API version on master
+
+We don't want to stop contributions to master just because we are releasing. At
+the same time, now that the release branch exists, we don't want API changes to
+go the now frozen API version.
+
+Create a new entry in `docs/sources/reference/api/` by copying the latest and
+bumping the version number (in both the file's name and content), and submit
+this in a PR against master.
```

Comment:
I think it also has to be added to [mkdocs.yml](https://github.com/docker/docker/blob/master/docs/mkdocs.yml#L152)

Or not yet at this stage?